acceptance/openstack/baremetal/v1/conductors_test.go

```diff
@@ -0,0 +1,35 @@
+//go:build acceptance || baremetal || conductors
+// +build acceptance baremetal conductors
+
+package v1
+
+import (
+	"testing"
+
+	"github.com/gophercloud/gophercloud/acceptance/clients"
+	"github.com/gophercloud/gophercloud/acceptance/tools"
+	"github.com/gophercloud/gophercloud/openstack/baremetal/v1/conductors"
+	"github.com/gophercloud/gophercloud/pagination"
+
+	th "github.com/gophercloud/gophercloud/testhelper"
+)
+
+func TestConductorsList(t *testing.T) {
+	clients.RequireLong(t)
+
+	client, err := clients.NewBareMetalV1Client()
+	th.AssertNoErr(t, err)
+	client.Microversion = "1.49"
+
+	err = conductors.List(client, conductors.ListOpts{}).EachPage(func(page pagination.Page) (bool, error) {
+		conductorList, err := conductors.ExtractConductors(page)
+		if err != nil {
+			return false, err
+		}
+
+		tools.PrintResource(t, conductorList)
```

Comment:
Sweet, I see there's one conductor returned.
https://github.com/gophercloud/gophercloud/actions/runs/5799867775/job/15720793946?pr=2722#step:5:1017

Could you also add tests for the Get()? Then it should be good to merge.

Reply:
Sure, no problem.